runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -1243,15 +1243,28 @@ class VM_JFRConstantPoolTypes {
 		/* Set CPU description */
 		OMRProcessorDesc desc = {};
 		omrsysinfo_get_processor_description(&desc);
-		char buffer[512];
-		omrsysinfo_get_processor_feature_string(&desc, buffer, sizeof(buffer));
-		UDATA len = strlen(buffer) + 1;
-		cpuInformation->description = (char *)j9mem_allocate_memory(len, OMRMEM_CATEGORY_VM);
-		if (NULL != cpuInformation->description) {
+		char smallbuf[1024];
+		char *buffer = smallbuf;
+		if (0 != omrsysinfo_get_processor_feature_string(&desc, buffer, sizeof(buffer))) {
+			int biggerSize = 8192;
```

Comment:
If we're sure 8192 bytes is enough, why not just change line 1246 and nothing else?

Reply:
I don't know if 8192 will always be enough, although I'd expect it to be sufficient for a long time. Do we want to allocate 8192 bytes on the stack, it seems too big. Really the OMR API isn't very nice, it should allocate a buffer or provide the needed size.

Reply:
Perhaps I should just change the value to 1024 for now, and remove the code to allocate a bigger buffer while investigating changing the OMR API?

Reply:
Updated it.